engines/bastion_katello/lib/bastion_katello/engine.rb

```diff
@@ -33,6 +33,7 @@ class Engine < ::Rails::Engine
         ),
         :config => {
           'consumerCertRPM' => consumer_cert_rpm,
+          'enable_deferred_download_policies' => !Foreman.in_rake? && Setting[:enable_deferred_download_policies],
```

Comment:
This is the reason @daviddavis is having to restart the application, this is defined at application boot time and not updated when the setting changes. I think your options would be:

 1) Change the value everytime the setting is itself is changed
 2) Query the Settings API from the UI to determine whether this is enabled
 3) Try to make this a proc or lambda to evaluate it at "runtime" aka when the value is sent up on page request